app/lib/job/job.dart

```diff
@@ -55,8 +55,12 @@ abstract class JobProcessor {
             _logger.info('$_serviceAsString job completed: $jobDescription');
           } catch (e, st) {
             status = JobStatus.failed;
-            _logger.severe(
-                '$_serviceAsString job error $jobDescription', e, st);
+
+            final str = '$e';
+            final isTransient = str.contains('The transaction was aborted.');
```

Comment:
There isn't a way to catch this exception a particular class?

Reply:
> There isn't a way to catch this exception a particular class?

Actually, we can :) PTAL.